README.md

```diff
@@ -1,3 +1,12 @@
+# Malcolm's Questions
+For #10, the test is looking for LastName to be correct.  I pulled in Name (which should contain LastName), but the test didn't like that.  When I switched Name out for LastName in the SOQL query, it worked fine.  Why wouldn't Name have pulled in LastName as well?
+
+For #12, do I really need to put in the 'expr0', 'expr1', etc... to pull out the aggregated expression?  Is there a more dynamic way or is this it?
+
+FOR #15, this just is looking for the youngest contacts??  Spent some time trying to figure out how to pull the contacts with the most recent birthdays (before realized this asked for birthdates).  Would be curious how to look for most recent birthdays, which would be more useful, maybe.
```

Comment:
That's a good question. This would involve looking at the month (and possibly day). There are SOQL date functions that could help with this, but it might involve some additional code to truly get what you're after. You might even have to add a new field to the Contact that holds the calendar day (starting with Jan 1 as 1). Then you can compare these numbers.